Modules/EthereumERC223Module.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types = 1);
+
+/*  Idea (c) 2023 Nikita Zhavoronkov, nikzh@nikzh.com
+ *  Copyright (c) 2023 3xpl developers, 3@3xpl.com, see CONTRIBUTORS.md
+ *  Distributed under the MIT software license, see LICENSE.md  */
+
+/*  This module processes ERC-223 token transfers in Ethereum. It requires either a geth or
+ *  an Erigon node to run. */
+
+final class EthereumERC223Module extends EVMERC223Module implements Module
+{
+    function initialize()
+    {
+        // CoreModule
+        $this->blockchain = 'ethereum';
+        $this->module = 'ethereum-erc-223';
+        $this->is_main = false;
+        $this->first_block_date = '2015-07-30';
+        $this->first_block_id = 0;
```

Comment:
@Har01d Not sure about `first_block_id`, we can set it on a block, when the ERC-223 was accepted by community or the block id where one of the first [ERC-223 contract](https://etherscan.io/address/0xcce968120e6ded56f32fbfe5a2ec06cbf1e7c8ed) was created, i.e. `18_691_377`, but I'm not really sure that it is really the first ERC-223 contract.